updater/internal/install/service_linux.go

```diff
@@ -0,0 +1,134 @@
+// Copyright  observIQ, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//go:build linux
+
+package install
+
+import (
+	"fmt"
+	"io"
+	"log"
+	"os"
+	"os/exec"
+	"path/filepath"
+)
+
+const linuxServiceName = "observiq-otel-collector"
+const linuxServiceFilePath = "/usr/lib/systemd/system/observiq-otel-collector.service"
+
+// newService returns an instance of the Service interface for managing the observiq-otel-collector service on the current OS.
+func newService(latestPath string) Service {
+	return linuxService{
+		newServiceFilePath:       filepath.Join(latestPath, "install", "observiq-otel-collector.service"),
+		serviceName:              linuxServiceName,
+		installedServiceFilePath: linuxServiceFilePath,
+	}
+}
+
+type linuxService struct {
+	// newServiceFilePath is the file path to the new unit file
+	newServiceFilePath string
+	// serviceName is the name of the service
+	serviceName string
+	// installedServiceFilePath is the file path to the installed unit file
+	installedServiceFilePath string
+}
+
+// Start the service
+func (l linuxService) Start() error {
+	//#nosec G204 -- serviceName is not determined by user input
+	cmd := exec.Command("systemctl", "start", l.serviceName)
+	if err := cmd.Run(); err != nil {
+		return fmt.Errorf("running systemctl failed: %w", err)
+	}
+	return nil
```

Comment:
This is fine but implies the updater is run as `root`. I think this should be fine since the collector currently runs as `root`. Not sure if we need to document this somewhere down the road.